docs/application/native/guides/app-management/ui-app/dali-widget-app.md

```diff
@@ -0,0 +1,269 @@
+# DALi Widget Application
+
+
+You can create a widget application with DALi to display small version of application on the home screen.
+
+To create a DALi widget application, you must:
+
+- Use the [Dali::WidgetApplication class](#widget-application-api).
+
+	> **Note**
+	> - The implementation of DALi widget application is similar to [DALi basic UI application](dali-basic-app.md), because the [Dali::WidgetApplication](../../../api/mobile/latest/classDali_1_1WidgetApplication.html) class inherits from the [Dali::Application](../../../api/mobile/latest/classDali_1_1Application.html) class.
+  > - Ensure you are familiar with the basic UI application details before tackling a widget application.
+
+- Ensure you start the event loop.
+
+- Implement your own widget class by inheriting the `Dali::Widget` class.
+
+
+**Figure: DALi widget application**
+
+![DALi widget application](./media/dali_widget_application.png)
```

Comment:
In our current checklist we have decided to add figure caption below the image.

Please add the **Figure: DALi widget application** caption below the image.